features/eolearn/features/feature_manipulation.py

```diff
@@ -257,11 +259,23 @@ def map_method(self, feature: np.ndarray, slope: float, intercept: float) -> np.
 class SpatialResizeTask(EOTask):
     """Resizes the specified spatial features of EOPatch."""
 
+    def _process_resize_parameters(
+        self, resize_parameters: Tuple[ResizeParam, Tuple[float, float]], bbox: BBox
+    ) -> Dict[str, Tuple[float, float]]:
+        resize_type, resize_type_param = resize_parameters
+        resize_type = ResizeParam(resize_type)
+
+        if resize_type == ResizeParam.RESOLUTION:
+            new_size = bbox_to_dimensions(bbox, resize_type_param)
+            return {ResizeParam.NEW_SIZE.value: new_size}
+
+        return {resize_type.value: resize_type_param}
+
     def __init__(
         self,
+        *,
+        resize_parameters: Tuple[ResizeParam, Tuple[float, float]],
         features: Any = ...,
```

Comment:
if we don't want to lock arguments to keyword-only (but keep `features=...`) the value `(ResizeParam.SCALE_FACTORS, (1, 1))` is technically a viable default. Not saying it's a good option, but maybe it's worth to be considered.

Reply:
agreed